src/components/homepage/homepagetemplate.js

```diff
@@ -42,13 +63,14 @@ class HomePage extends React.Component {
 		/* eslint-disable react/jsx-key */
 		return [
 			<StyleHeader />,
```

Comment:
Now there is no need to return `StyleHeader` and therefore no need to return an array here now. Just the `div`.